### ISSUE TITLE
FE: UX/UI improvements

### ISSUE THREAD

**Issue by SzymonSt** [SzymonSt/docker-signalone#107]
## Functional

- [x]  **Google login blocker**_(Critical)_

Google login flow must open log in screen in same tab without pop up as docker extension container does not allow that by default:

![Image](https://github.com/SzymonSt/docker-signalone/assets/48925621/85305354-2467-4895-a4fe-816ee2fb600b) 

- [x] **Settings icon bug**_(Critical)_

 Settings icon doesn't show up after log in, page needs to be refreshed for it to show up.
after login:

![Image](https://github.com/SzymonSt/docker-signalone/assets/48925621/86d7491a-3fd2-4599-9594-067848ee5c99)

after refresh while logged in:

![Image](https://github.com/SzymonSt/docker-signalone/assets/48925621/98cb38e9-a863-4ebf-88fc-b9c5e69961c7)

- [x] Issues loader

Loader shouldn't show up every 15seconds while issues are being pulled. It is causing a flicker effect and distracts while investigating issue

- [x] Show/Hide side menu

Currently, the user can't hide the side menu with filters and an issue list. It would be nice to provide some button which after clicking would hide/open the side menu. It will save some valuable space and allow the user to focus only on the selected issue.

Jira has a good UX for that (little arrow button on the edge of the side menu)

![Image](https://github.com/SzymonSt/docker-signalone/assets/68923217/8a286b54-8c1b-467c-b2a2-8b8241989435)

- [x] Create an `empty` component for a detailed issue view 
Currently, we don't display anything until the user selects an issue. We should add some message (maybe with some hero image) that will instruct the user to select an issue from the list

## Visual improvements/fixes

- [x] Issue detailed view - poor responsiveness
Issue: Currently if the user resizes the window with an open detailed issue view the container doesn't adopt to the new dimensions. 

https://github.com/SzymonSt/docker-signalone/assets/68923217/25a42e5a-6e29-4e29-84dd-c79b3d9f680d

- [x] Settings - user agent message
Issue: After enabling the agent user gets an unfriendly message in the toast message.
Suggestion: Provide some better verbiage for the toast message for the agent state.

- [x] Issue list - empty component is missing
Issue: When a user doesn't have the issues, we should hide the list controls and display an `empty` component which will inform the user that she/he does not have any issues.

![Image](https://github.com/SzymonSt/docker-signalone/assets/68923217/4c817ab2-58ea-4f86-8bb8-bd8ee40cc373)

## Keyboard interaction/styling

- [x] Keyboard focus styling - clear button
Issue: If the user focuses the `clear` button with the keyboard it changes the text color to white and the user can't see what is written on the button
Suggestion: Provide styling for the `:focus-visible` pseudo-class

![Image](https://github.com/SzymonSt/docker-signalone/assets/68923217/e9e7dcdc-e6bb-4889-9a7a-0bd3c856df03)
